test/unit/integrations/payu_in/payu_in_return_test.rb

```diff
@@ -16,11 +20,16 @@ def test_success
     assert_equal 'Completed', @payu.status('4ba4afe87f7e73468f2a','10.00')
   end
 
-  def test_failure_is_successful
+  def test_pending_is_successful
```

Comment:
Should be `test_failed_return_is_a_failed_status`

Reply:
doing a commit amend just to keep it short. Hope you don't mind.